include/cell/copy/global_to_register.hpp

```diff
@@ -126,7 +133,24 @@ struct GlobalToRegLoaderImpl<Global_, Reg_, kRowExec_, kColExec_,
     static constexpr int kRowExec = kRowExec_;
     static constexpr int kColExec = kColExec_;
 
-    DEVICE void operator()(const DType* src, Reg& dst) {}
+    DEVICE void operator()(const DType* src, Reg& dst) {
+        int lane_id = threadIdx.x % warpSize;
+
+        const DType* data;
+
+#pragma unroll
+        for (int i = 0; i < kColExec; ++i) {
+            int col = i * kTileSize + lane_id / 4;
+            for (int j = 0; j < kRowExec; ++j) {
+                int row = j * kTileSize + (lane_id % 4) * 2;
+                data = src + col * Global::kColStride + row;
+                using Loader = GlobalToRegMatLoader<Global, BaseTile,
+                                                    tl::Layout::ColMajor>;
+                Loader loader;
```

Comment:
Move lines 147 to 149 outside of the loop; otherwise, the loader in line 149 will be constructed and destroyed multiple times, which is unnecessary.

Reply:
fixed.